cluster-logging/clo-og.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: ClusterLogging.v1.logging.openshift.io
+  generateName: openshift-logging-
+  generation: 1
```

Comment:
may I know why do you specify the `generation` here and in `cluster-logging/clo-sub.yaml ` CR? It's not needed - this is managed by k8s

Reply:
because it's a left-over :| I initially installed the operator from the UI, then exported the CR from the CLI, but I forgot to remove this field 🤦‍♂

Reply:
removed in 8236cd8